### PR TITLE
upgrade to webpacker 6

### PR DESCRIPTION
**Pull request by thomascchen**
upgrade webpacker from the version 5 to 6

### CODE REVIEW

**Comment by thomascchen** [comment]
@davekaro want to take another look? 😄

I deployed this branch on Heroku, and it worked this time. I realized that I must have run `yarn upgrade @rails/webpacker --latest` in #57 because the yarn.lock actually had 6.0.0-pre.1 specified instead of 6.0.0.beta.2: https://github.com/codeforUV/food_resources/pull/57/files#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2deR863

Didn't run that on this branch, which I think is the only difference.

File: package.json [hunk 1]
```diff
@@ -8,15 +8,30 @@
...
     "@rails/activestorage": "^6.0.0",
     "@rails/ujs": "^6.0.0",
-    "@rails/webpacker": "5.2.1",
+    "@rails/webpacker": "^6.0.0-beta.2",
     "@tailwindcss/postcss7-compat": "^2.0.2",
     "autoprefixer": "^9",
+    "css-loader": "^5.0.1",
+    "css-minimizer-webpack-plugin": "^1.1.5",
+    "mini-css-extract-plugin": "^1.3.3",
     "postcss": "^7",
```
Comment (davekaro) [inline package.json:17]: postcss 7?!

**Review by davekaro — APPROVED** [review]
🌮

File: package.json (hunk 1 above)
Comment (davekaro) [reply package.json:17]: Nevermind! I see https://github.com/codeforUV/food_resources/pull/60

**Pull request merged**